lib/YaST/Bootloader/BootloaderSettingsController.pm

```diff
@@ -10,9 +10,11 @@ package YaST::Bootloader::BootloaderSettingsController;
 use strict;
 use warnings;
 use YuiRestClient;
+use testapi;
 use YaST::Bootloader::BootCodeOptionsPage;
 use YaST::Bootloader::KernelParametersPage;
 use YaST::Bootloader::BootloaderOptionsPage;
+use YaST::workarounds;
```

Comment:
it is not a workaround, the bug is fixed, it is just another testing path that we should cover to avoid regression, but we just need to mention that in a comment in the method.